README.md

```diff
@@ -487,7 +487,7 @@ let's add some pizzazz! ✨
 Open the `mix.exs` file and add `:tailwind` to `defp deps`:
 
 ```elixir
-{:tailwind, "~> 0.1", runtime: Mix.env() == :dev},
+{:tailwind, "~> 0.1.9", runtime: Mix.env() == :dev},
```

Comment:
FYI: `~> 0.1` means "get the latest version equal to or greater than version 0.1"
So `0.1.9` is _already_ in the `mix.lock`:

https://github.com/dwyl/learn-tailwind/blob/089fc392a364cecb5bde82531655b3b2a081b4b4/mix.lock#L31

But doesn't _hurt_ to bump this in the `README.md`. 👌

Reply:
👌